### PR TITLE
swagger: add kubeconfig download

### PR DESCRIPTION
**Pull request by avishayt**

### CODE REVIEW

**Comment by filanov** [comment]
@eranco74 can you please remind me where does this file used?

File: swagger.yaml [hunk 1]
```diff
@@ -174,6 +174,31 @@ paths:
...
+      produces:
+        - text/x-yaml
+      parameters:
+        - in: path
+          name: clusterId
+          description: The ID of the cluster whose kubeconfig to download
+          type: string
+          format: uuid
+          required: true
+      responses:
```
Comment (ronniel1) [inline swagger.yaml]: What will be the response if the kubeconfig file was not created yet?

Comment (filanov) [reply swagger.yaml]: i think that it's a synchronous API.

Comment (avishayt) [reply swagger.yaml]: If no kubeconfig it should be 409

**Pull request closed without merging**